src/coreclr/jit/emitarm64.cpp

```diff
@@ -25174,17 +25178,24 @@ BYTE* emitter::emitOutput_InstrSve(BYTE* dst, instrDesc* id)
         }
 
         case IF_SVE_EB_1A: // ........xx...... ..hiiiiiiiiddddd -- SVE broadcast integer immediate (unpredicated)
-        case IF_SVE_EC_1A: // ........xx...... ..hiiiiiiiiddddd -- SVE integer add/subtract immediate (unpredicated)
-        {
             imm  = emitGetInsSC(id);
             code = emitInsCodeSve(ins, fmt);
             code |= insEncodeReg_V_4_to_0(id->idReg1());                  // ddddd
+            code |= insEncodeElemsize(optGetSveElemsize(id->idInsOpt())); // xx
```

Comment:
Isn't `IF_SVE_EB_1A` and `IF_SVE_EC_1A` cases same?

Reply:
Yes, I must've moved this when refactoring and forgot to put it back; I'll fix this